### PR TITLE
[#675] fix: filter no space exception in checkStorageReadAndWrite

### PR DESCRIPTION
**Pull request by xumanbu**
### What changes were proposed in this pull request?

Filter no space exception in checkStorageReadAndWrite

### Why are the changes needed?

Fix: #675 

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
UT

### CODE REVIEW

File: server/src/main/java/org/apache/uniffle/server/LocalStorageChecker.java [hunk 1]
```diff
@@ -73,7 +73,8 @@ public boolean checkIsHealthy() {
     int corruptedDirs = 0;
 
     for (StorageInfo storageInfo : storageInfos) {
-      if (!storageInfo.checkStorageReadAndWrite()) {
+      if (storageInfo.checkIsSpaceEnough()
```
Comment (jerqi) [inline server/src/main/java/org/apache/uniffle/server/LocalStorageChecker.java:76]: `checkIsSpaceEnough` can't guaratee that the disk is full. It only means that the disk capacity is very high.

**Comment by jerqi** [comment]
Could you test this patch by hand? Could you modify the description of `What changes were proposed in this pull request?`?

File: server/src/main/java/org/apache/uniffle/server/LocalStorageChecker.java [hunk 2]
```diff
@@ -200,6 +202,9 @@ boolean checkStorageReadAndWrite() {
         }
       } catch (Exception e) {
         LOG.error("Storage read and write error. Storage dir: {}", storageDir, e);
+        if(e.getMessage()!= null && DEVICE_NO_SPACE_ERROR_MESSAGE.equals(e.getMessage())){
```
Comment (jerqi) [inline server/src/main/java/org/apache/uniffle/server/LocalStorageChecker.java:205]: Could you add some comments to explain why we do so?

**Review by jerqi — COMMENTED** [review]
You need modify the code style. The code style check failed.

**Comment by zuston** [comment]
Nice catch. I found the same problems. 👍

**Pull request merged**